app/src/main/java/mysh/dev/gemcap/ui/components/dialogs/IdentityImportDialog.kt

```
@@ -0,0 +1,392 @@
+
+/**
+ * Dialog for importing a client identity from a PEM file.
+ *
+ * Flow:
+ * 1) User picks file.
+ * 2) PEM is parsed (and passphrase requested if required).
+ * 3) Duplicate fingerprint check runs.
+ * 4) User confirms import (or replace on conflict).
+ */
+@Composable
+fun IdentityImportDialog(
```

detekt (1.23.8)

[warning] The function IdentityImportDialog appears to be too complex based on Cyclomatic Complexity (complexity: 33). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CyclomaticComplexMethod)